uint16[] src_robot_ids
uint16[] dst_robot_ids
uint32[] src_pose_ids
uint32[] dst_pose_ids
float64[] weights